src/cli/bin.js

```diff
@@ -41,18 +38,15 @@ async function main (args) {
       }
       process.exit(1)
     })
-    .finally(async () => {
-      // If an IPFS instance was used in the handler then clean it up here
-      if (getIpfs && getIpfs.instance) {
-        try {
-          const cleanup = getIpfs.rest[0]
-          await cleanup()
-        } catch (err) {
-          debug(err)
-          process.exit(1)
-        }
-      }
-    })
+
+  onExit(() => {
+    // If an IPFS instance was used in the handler then clean it up here
+    if (getIpfs && getIpfs.instance) {
+      const cleanup = getIpfs.rest[0]
+
+      cleanup.catch(err => debug(err))
+    }
+  })
```

Comment:
I don't think we're able to do async operations on process exit? Do we need a synchronous cleanup instead?